### PR TITLE
blog: add pagination

### PR DESCRIPTION
**Pull request by nicks**
Hello @hyu,

Please review the following commits I made in branch nicks/paginate:

1477de2604277af36490260118e07e2732be877a (2020-04-28 17:31:10 -0400)
blog: add pagination

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
Here's what it looks like:

Screenshot from 2020-04-28 17-31-47

I'm hoping that this will help with SEO a bunch, so that we don't have a ~30MB blog index

File: blog/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: Tilt Blog
```
Comment (nicks) [inline blog/index.html:2]: fyi -- this file is mostly unchanged -- the pagination plugin just needs the file to be called index.html

File: src/_sass/layout.scss [hunk 2]
```diff
@@ -1423,3 +1423,23 @@ a.attached-above {
   font-weight: bold;
   margin-top: - $spacing-unit / 2
 }
+
+.pagination {
+  display: flex;
+  margin: - $spacing-unit auto $spacing-unit auto;
```
Comment (hyu) [inline src/_sass/layout.scss:1429]: When the pagination appears at the bottom of the page, it's a little too close to the blog excerpts and can get visually lost. If we leave more room then people are less likely to miss this additional content.

```suggestion
  margin: $spacing-unit auto $spacing-unit auto;
```

Comment (nicks) [reply src/_sass/layout.scss]: agreed, i added more spacing to the bottom version

**Pull request merged**